dataverk_airflow/kubernetes_operator.py

```diff
@@ -112,7 +112,7 @@ def on_failure(context):
         working_dir = POD_WORKSPACE_DIR
 
     if requirements_path:
-        cmds = ["pip", "install", "-r", f"{POD_WORKSPACE_DIR}/{requirements_path}", "--user", "&&"] + cmds
+        cmds = [f"pip install -r {POD_WORKSPACE_DIR}/{requirements_path} --user"] + cmds
```

Comment:
Åja, jeg skjønte ikke at vi gikk bort fra dette som en liste. Snodig at vi må det. Er dette pga. joinen lengre ned?

Reply:
Ser faktisk at k8s anbefaler lignende det du gjør.
https://kubernetes.io/docs/tasks/inject-data-application/define-command-argument-container/#run-a-command-in-a-shell
```
command: ["/bin/sh"]
args: ["-c", "while true; do echo hello; sleep 10;done"]
```